public/js/main.js

```diff
@@ -44,18 +44,16 @@ function render(){
   showFooter();
 }
 
-function Task(content){
-  this.id = new Date().valueOf();
-  this.content = content;
-  this.done = false;
-}
 // add new task
 var todoInput = document.getElementById('todo-input')
 todoInput.onkeypress = function (e){
   const enterKey = 13;
   todoInput = document.getElementById('todo-input');
   if(e.charCode === enterKey && todoInput.value.trim()!==''){
-    let newTask = new Task(todoInput.value);
+    let newTask = {};
```

Comment:
We won't assign the value to `newTask` again, so make it as a const

- [ ] use `const`